src/twitch/routers.py

```diff
@@ -0,0 +1,100 @@
+import json
+from typing import Annotated
+
+from fastapi import APIRouter, Depends
+
+from brokers.producer import producer
+from cache import RedisCacheManager
+from core.enums import ObjectStatus
+from db import get_twitch_database
+from db.database_managers import TwitchDatabaseManager
+from dependecies import get_cache_manager
+from schemas import TwitchResponseFromParser
+
+from .config import TwitchSettings
+from .dependencies import get_twitch_parser
+from .service import TwitchParser
+
+twitch_router = APIRouter(prefix="/twitch")
+
+
+TwitchParserObject = Annotated[TwitchParser, Depends(get_twitch_parser)]
+TwitchDb = Annotated[TwitchDatabaseManager, Depends(get_twitch_database)]
+CacheMngr = Annotated[RedisCacheManager, Depends(get_cache_manager)]
+settings = TwitchSettings()
+
+
+@twitch_router.post("/streams")
+async def parse_streams(
+    parser: TwitchParserObject,
+    db: TwitchDb,
+    cache: CacheMngr,
+    first: int = 10,
+    game_id: int | None = None,
+    language: str = "en",
+):
+    """This views stands for parsing streams, processed streams are saved to cache and db"""
+
+    query_params = {"first": first, "language": language}
```

Comment:
what is the 'first' argument for? 
is it offset? like first 10 elements? then name it offset

Reply:
It is argument as it used in twitch api, so they have their doc says "The maximum number of items to return per page in the response", so i just copied their name from it